### DISCUSSION TITLE
Error: Mac OS install Loop

### DISCUSSION THREAD

**Discussion by finfrig** [publicsamples/Quetzalcoatl#28] [General]
I'm getting the install loop problem as listed by someone else previously. I point Quetz to the hr1 file, it says installed successfully and to restart, only to be prompted with the same install upon re-launch.

Ive tried moving the hr1 file off of downloads to desktop, using a custom samples folder, but no luck. I'm trying to install .8, on Mac Ventura and with Ableton 12. Really hope I can use this lovely synth soon, I will try to install an older version.

**Comment by finfrig** [comment]
I just noticed that Quetz is asking me to select and install "Quetzalcoatl Resources 1.0.0.h1 file" and on the .8 download page, it has the 0.7.1.hr1 release file, which is what I am using to install the samples. Could this be the problem?

**Comment by RickTaylor79** [comment]
Take a look at 0.8.1,  it removes the sample install dialogue entirely.

https://github.com/publicsamples/Quetzalcoatl/releases/tag/0.8.1
